docs/root/modules/entra/config.md

```diff
@@ -12,9 +12,36 @@ To set up the Entra client,
 1. Go to [App Registrations](https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) in the Azure portal
 1. Create a new app registration.
 1. Grant it the following permissions:
+    - `Group.Read.All`
+        - Read all groups
+        - Type: Application
     - `AdministrativeUnit.Read.All`
+        - Read all administrative units
+        - Type: Application
     - `Directory.Read.All`
+        - Read directory data
+        - Type: Application
+    - `User.Read.All`
+        - Read all users' full profiles
+        - Type: Application
+    - `GroupMember.Read.All`
+        - Read all group memberships
+        - Type: Application
+    - `AppRoleAssignment.ReadWrite.All`
```

Comment:
Does this need the write or are we good with just read?

Reply:
<img width="838" height="127" alt="Screenshot 2025-07-24 at 3 07 12 PM" src="https://github.com/user-attachments/assets/24f4695e-47f0-46b6-8d76-5d3d75dc2386" />
Unfortunately they don't have a readonly perm for AppRoleAssignment